app-office/sc-im/sc-im-0.8.2.ebuild

```diff
@@ -0,0 +1,143 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-1 luajit )
+
+inherit lua-single toolchain-funcs flag-o-matic
+
+DESCRIPTION="Spreadsheet Calculator Improvised -- An ncurses spreadsheet program for terminal"
+HOMEPAGE="https://github.com/andmarti1424/sc-im"
+SRC_URI="https://github.com/andmarti1424/sc-im/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-4"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="X plots wayland xls xlsx lua ods tmux"
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )"
+
+DEPEND="
+	sys-libs/ncurses
+
+	lua? (
+		${LUA_DEPS}
+	)
+	ods? (
+		dev-libs/libxml2
+		dev-libs/libzip
+	)
+	plots? ( sci-visualization/gnuplot )
+	tmux? ( app-misc/tmux )
+	wayland? ( gui-apps/wl-clipboard )
+	X? ( x11-misc/xclip )
+	xls? (
+		dev-libs/libxls
+	)
+	xlsx? (
+		dev-libs/libxlsxwriter
+		dev-libs/libxml2
+		dev-libs/libzip
+	)
+"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+S="${WORKDIR}/${P}/src"
+
+pkg_setup() {
+	CONFLICTING=$(usex tmux "tmux " "")$(usex wayland "wayland " "")$(usex X "X" "")
+	if ( use tmux && ( use wayland || use X ) ) ; then
+		elog "Conflicting flags for clipboard support are set: ${CONFLICTING}"
+		elog "tmux support has been preferred."
+	elif ( use wayland && use X ) ; then
+		elog "Conflicting flags for clipboard support are set: ${CONFLICTING}"
+		elog "Wayland support has been preferred."
+	fi
+
+	# Run lua setup
+	lua-single_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	# Clean Makefile from all sorts of flag / lib setting
+	sed -i -e '/CFLAGS +=\|LDLIBS +=/d' Makefile \
+		|| die "sed fix failed. Uh-oh..."
+	# Also clean the now useless comments and logic
+	sed -i -e '/#\|if\|else/d' Makefile \
+		|| die "sed fix failed. Uh-oh..."
+}
+
+src_configure() {
+	tc-export CC
+
+	PKGCONF=$(tc-getPKG_CONFIG)
```

Comment:
I wonder -- why not just do: `tc-export PKG_CONFIG` (you can add it on to the previous tc-export line) and use ${PKG_CONFIG} everywhere instead?

Reply:
I had just taken that from the part about directly calling pkg-config from the devmanual and never thought about exporting it with CC, I think I need to read up on toolchain-funcs. Thank you and thank you for all the other comments.